crates/astria-composer/src/utils.rs

```diff
@@ -2,11 +2,9 @@ use astria_eyre::eyre::Report;
 use tracing::{
     error,
     info,
-    instrument,
 };
 
-#[instrument(skip_all)]
-pub(crate) fn report_exit_reason(reason: &Result<&str, Report>) {
+pub(crate) fn report_exit_reason(reason: Result<&&str, &Report>) {
```

Comment:
Using `Result::as_deref` above this can become:
```suggestion
pub(crate) fn report_exit_reason(reason: Result<&str, &Report>) {
```